requirements-dev.txt

```diff
@@ -0,0 +1,3 @@
+tox==3.20.0
```

Comment:
Thank you @michaelloewensteinviz ! Could be great that the `requirements-dev.txt` had this at first line `-r requirements.txt` to install dev dependencies and the rest of dependencies at once

Reply:
@avara1986 , not sure what you mean, having all the dependencies in a single file (requirements.txt) ?

Reply:
You can see [this file](https://github.com/avara1986/pivoandcode-2019-11-15/blob/master/chat_front/requirements-docker.txt) as example of what a I mean :) 

If you add in your file `requirements-dev.txt` this: `-r requirements.txt` when you type `pip install -r requirements-dev.txt` pip will install requirements-dev AND requirements at once

Reply:
Good point! added